### DISCUSSION TITLE
Enabling spectre mitigation in consumed packages

### DISCUSSION THREAD

**Discussion by adrien-chaussabel** [microsoft/vcpkg#38434] [Q&A]
I'm currently consuming 2 packages (clipper2 and zxing-cpp) through vcpkg that raise errors when running BinSkim due to not being compiled with the /Qspectre flag. Looking for advice on how to get spectre mitigation enabled.

**Comment by Osyotr** [comment]
https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_cxx_flags
